pkg/provisioningv2/rke2/installer/server.go

```diff
@@ -0,0 +1,28 @@
+package installer
+
+import (
+	"net/http"
+)
+
+type handler struct{}
+
+var Handler *handler
+
+func (s *handler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
+	var err error
+	var content []byte
+
+	switch req.URL.Path {
+	case SystemAgentInstallPath:
+		content, err = LinuxInstallScript("", nil, req.Host)
+	case WindowsRke2InstallPath:
+		content, err = WindowsInstallScript("", nil, req.Host)
+	}
```

Comment:
Does this need a base case?

Reply:
if other urls were added to handle, not possible to get here beyond the two passed... who would ever get here and serve the response? even if it did it'd send an empty string back in plain text, not the end of the world